### ISSUE TITLE
plot_coordinates fails when matplotlib figure object is provided as an argument

### ISSUE THREAD

**Issue by ma-sadeghi** [PMEAL/OpenPNM#1470]
# How to reproduce the bug
```
import openpnm as op
import matplotlib.pyplot as plt

net = op.network.Cubic(shape=[5, 5, 5])
fig, ax = plt.subplots()
op.topotools.plot_coordinates(network=net, fig=fig)
```

**Comment by jgostick** [comment]
Hmmm, that never used to happen, but maybe it's because your using subplots?  Try sending a normal single figure.

**Comment by jgostick** [comment]
we actually have a notebook example that uses this features, but it's not with subplots:

https://github.com/PMEAL/OpenPNM/blob/master/examples/topology/Using%20Queries%20-%20Level%201.ipynb

**Comment by ma-sadeghi** [comment]
It's passing `fig` for the first time that's causing the bug. In the example, we first assign the output of `plot_coordinates` to `fig` and then pass `fig` in the later calls to `plot_coordinates`.

**Comment by ma-sadeghi** [comment]
This is now addressed in #1475
